modules/serverless-common/pages/service-mesh/common-service-mesh-network-isolation.adoc

```diff
@@ -0,0 +1,438 @@
+----
++
+And apply the generated resources to your cluster:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
++
+[NOTE]
+====
+The helm chart has several options that can be passed to configure the generated resources. Please refer to the link:https://github.com/openshift-knative/knative-istio-authz-chart/blob/main/values.yaml[values.yaml] for a full reference.
+====
+
+
+.Verifying the configuration
+
+This verification is assuming that we have two tenants with one namespace each, all part of the `ServiceMeshMemberRoll`, configured with resources listed above.
+We can then use curl to verify the connectivity:
+
+. Deploy Knative Services in both tenants namespaces and a `curl` pod to run test commands:
++
+[source,terminal]
+----
+# Tenant 1
+cat <<-EOF | oc apply -f -
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: test-webapp
+  namespace: tenant-1
+  annotations:
+    serving.knative.openshift.io/enablePassthrough: "true"
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: 'true'
+        autoscaling.knative.dev/target-burst-capacity: "-1"
+    spec:
+      containers:
+        - image: docker.io/openshift/hello-openshift
+          env:
+            - name: RESPONSE
+              value: "Hello Serverless!"
+EOF
+
+cat <<-EOF | oc apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: curl
+  namespace: tenant-1
+  labels:
+    app: curl
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: curl
+  template:
+    metadata:
+      labels:
+        app: curl
+      annotations:
+        sidecar.istio.io/inject: 'true'
+    spec:
+      containers:
+      - name: curl
+        image: curlimages/curl
+        command:
+        - sleep
+        - "3600"
+EOF
+
+# Tenant 2
+cat <<-EOF | oc apply -f -
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: test-webapp
+  namespace: tenant-2
+  annotations:
+    serving.knative.openshift.io/enablePassthrough: "true"
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: 'true'
+        autoscaling.knative.dev/target-burst-capacity: "-1"
+    spec:
+      containers:
+        - image: docker.io/openshift/hello-openshift
+          env:
+            - name: RESPONSE
+              value: "Hello Serverless!"
+EOF
+----
+
+. Verification
++
+[source,terminal]
+----
+# Test tenant-1 -> tenant-1 via cluster local domain (allowed)
+oc exec deployment/curl -n tenant-1 -it -- curl -v http://test-webapp.tenant-1:80
+
+HTTP/1.1 200 OK
+content-length: 18
+content-type: text/plain; charset=utf-8
+date: Wed, 26 Jul 2023 12:49:59 GMT
+server: envoy
+x-envoy-upstream-service-time: 9
+
+Hello Serverless!
+
+
+# Test tenant-1 -> tenant-1 via external domain (allowed)
+EXTERNAL_URL=$(oc get ksvc -n tenant-1 test-webapp -o custom-columns=:.status.url --no-headers)
+oc exec deployment/curl -n tenant-1 -it -- curl -ik $EXTERNAL_URL
```

Comment:
If we can assume that `kn` is installed we could simplify this to 

```suggestion
oc exec deployment/curl -n tenant-1 -it -- curl -ik $(kn service describe test-webapp -o url -n tenant-1)
```

Reply:
I would prefer to discuss the use `kn` as a separate topic. If we do this, we should do it over all the docs.